doc/Changelog.md

```diff
@@ -4,6 +4,7 @@ Changelog {#Changelog}
 # master {#master}
 
 * [#313](https://github.com/BlueBrain/Livre/pull/313):
+  Camera synchronization is working.
   Replaced the data bounding box by the data to Livre spaces 
```

Comment:
What do you mean by Livre space ? It's not only livre space, RTNeuron camera is also synchronized.